### DISCUSSION TITLE
Possible to access files, e.g. zip files to install repos?

### DISCUSSION THREAD

**Discussion by Has3n** [cguZZman/plugin.googledrive#224] [Q&A]
I needed a solution to access zip files from within kodi, so I thought I use gdrive to get them into kodi. But it seems it only supports pictures and video streaming? When accessing “my drive” I see no content. Is this not the right tool for my purposes? Thanks:)

**Comment by cguZZman** [comment]
That was the reason for the source mode in the addon. But due to changes in the API from Google, it is not available as of now.

But it is working for [OneDrive](https://forum.kodi.tv/showthread.php?tid=228443), available in the Kodi Repository or my repository.
